servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -579,21 +591,24 @@ private TraversalResult traverseToInode(String[] pathComponents, boolean persist
      */
     private final Inode mInode;
 
-    private final List<Inode> mPersisted;
+    /**
+     * A list of existing Inodes that are found to be persisted during traversal.
```

Comment:
Rename the variable to `mNonPersisted` and change the comment to:

`A list of non-persisted inodes encountered by the traversal.`